knack/parser.py

```diff
@@ -174,6 +175,7 @@ def load_command_table(self, command_loader):
                 param.deprecate_info = arg.deprecate_info
                 param.preview_info = arg.preview_info
                 param.experimental_info = arg.experimental_info
+                param.default_from = arg.default_from
```

Comment:
I prefer calling it `default_value_source` for better understanding.

Reply:
thanks, updated.